### PR TITLE
Pin pipenv in Dockerfile

### PR DESCRIPTION
**Pull request by stephendonner**
@davehunt r?

Passing ad-hoc here: https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/mozillians.adhoc/221/console

### CODE REVIEW

**Comment by stephendonner** [comment]
@davehunt here and in other repos, we might want to leave ```--skip-lock``` in; if we're not using the Pipfile.lock due to the known issues with @pyup-bot auto-managing dependencies (not updating the meta hashes), then why go through:

```
23:23:12 Step 14 : RUN pipenv install --system
23:23:12  ---> Running in 219f06ce5d95
23:23:13 Pipfile.lock not found, creating…
23:23:13 Locking [dev-packages] dependencies…
23:23:13 Locking [packages] dependencies…
23:23:18 Updated Pipfile.lock (c5a696)!
23:23:18 Installing dependencies from Pipfile.lock (c5a696)…
```

Isn't that just an extraneous, intermediary step, then?

**Comment by davehunt** [comment]
> we might want to leave `--skip-lock` in

fair enough, good point!

File: pipenv.txt [hunk 1]
```diff
@@ -1 +1 @@
-pipenv==11.1.3
+pipenv==10.1.2
```
Comment (davehunt) [inline pipenv.txt]: Why are we rolling this back?

File: Dockerfile [hunk 2]
```diff
@@ -34,7 +37,7 @@ RUN curl -fsSLo /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/r
 
 WORKDIR /src
 COPY Pipfile /src/
-RUN pipenv install --system --skip-lock
+RUN pipenv install --system
```
Comment (davehunt) [inline Dockerfile]: You convinced me, let's keep this.

File: pipenv.txt (hunk 1 above)
Comment (stephendonner) [reply pipenv.txt]: With 11.1.3, I was getting errors as I removed --skip-lock and --deploy.  I didn't want to argue with Kenneth in the issue I filed, but he attested it should work...

**Comment by stephendonner** [comment]
Thanks; left ```--skip-lock``` back in, and re-ran: https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/mozillians.adhoc/227/console

File: pipenv.txt (hunk 1 above)
Comment (stephendonner) [reply pipenv.txt]: @davehunt see e.g. https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/go-bouncer.adhoc/152/console

**Pull request merged**